tests/topotests/all_protocol_startup/test_all_protocol_startup.py

```diff
@@ -1713,6 +1714,105 @@ def _show_func():
     net["r1"].cmd('vtysh -c "conf" -c "no nexthop-group resilience"')
 
 
+def test_interface_stuff():
+    global fatal_error
+    net = get_topogen().net
+
+    # Skip if previous fatal error condition is raised
+    if fatal_error != "":
+        pytest.skip(fatal_error)
+
+    print("\n\n** Verifying some interface code")
+    print("************************************\n")
```

Comment:
can you use `step()` instead?

Reply:
I’m not interested in switching the test over to step.